app/controllers/companies_controller.rb

```diff
@@ -0,0 +1,9 @@
+class CompaniesController < ApplicationController
+  # get '', to: 'companies#company', constraints: { subdomain: /.+/ }
+  def company
```

Comment:
what is the use of this method?

Reply:
This is just the home page for company, just for testing scopes etc.

Reply:
then it should be index page not the company. Can you please change this one?

Reply:
done

Reply:
But well it was just a page for testing it should be replaced after merge with develop